### PR TITLE
Use <script>-based API in the extension #670

### PR DESCRIPTION
**Pull request by hayatoito**

### CODE REVIEW

File: explainers/subresource-loading-opaque-origin-iframes.md [hunk 1]
```diff
@@ -56,11 +56,12 @@ Suppose that the bundle, `subresources.wbn`, includes the following resources:
...
 ```html
-<link
-  rel="webbundle"
-  href="https://example.com/dir/subresources.wbn"
-  resources="uuid-in-package:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"
-/>
+<script type="webbundle">
+{
+  source: "https://example.com/dir/subresources.wbn",
+  resources: ["uuid-in-package:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"],
```
Comment (irori) [inline explainers/subresource-loading-opaque-origin-iframes.md:62]:
```suggestion
  resources: ["uuid-in-package:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"]
```

File: explainers/subresource-loading-opaque-origin-iframes.md [hunk 2]
```diff
@@ -96,11 +97,13 @@ the page can load `uuid-in-package` resources in web bundles served from
...
-<link rel="webbundle"
-  href="https://cdn.example/subresources.wbn"
-  resources="uuid-in-package:429fcc4e-0696-4bad-b099-ee9175f023ae
-             uuid-in-package:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"
-/>
+<script type="webbundle">
+{
+  source: "https://cdn.example/subresources.wbn",
+  resources: ["uuid-in-package:429fcc4e-0696-4bad-b099-ee9175f023ae",
+              "uuid-in-package:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"],
```
Comment (irori) [inline explainers/subresource-loading-opaque-origin-iframes.md:104]:
```suggestion
              "uuid-in-package:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"]
```

**Review by irori — APPROVED** [review]
lgtm with JSON syntax nits

File: explainers/subresource-loading-opaque-origin-iframes.md (hunk 1 above)
Comment (hayatoito) [reply explainers/subresource-loading-opaque-origin-iframes.md]: Done

File: explainers/subresource-loading-opaque-origin-iframes.md (hunk 2 above)
Comment (hayatoito) [reply explainers/subresource-loading-opaque-origin-iframes.md]: Done

**Comment by hayatoito** [comment]
Thanks! Fixed JSON syntex.

**Pull request merged**